.github/workflows/default.yml

```diff
@@ -84,7 +93,7 @@ jobs:
       - uses: actions/download-artifact@v2
         with:
           name: "mips kernel+ramdisk"
-      - run: ./run_tests.py --board malta --timeout=80 --times=50
+      - run: ./run_tests.py --board malta --timeout=60 --times=50
```

Comment:
Is it magically faster now?

Reply:
No. OTOH I don't know why it was so high compared to AArch64, so I lowered it a bit.

Reply:
See 4fe61bea98451a38898e323506841e1fe0e8338c

Reply:
If you look into tests runtime AArch64 & MIPS take roughly 30 minutes and 45 minutes for RISC-V. Timeouts seem to be set up properly.

In fact the only way to set up timeouts in more reasonable way would be to measure time taken by each test. So test framework should log for each test and each testrun how much time did it take. Then we could make well informed decision.